### ISSUE TITLE
Coordinate Conversion doesn't calculate correct output coordinate for GARS 15' x 15' minute cell coordinate

### ISSUE THREAD

**Issue by elinz** [Esri/solutions-webappbuilder-widgets#779]
### Widget
Coordinate Conversion
### Version of widget
11 Jan 2017 18:14
### Bug or Enhancement

According to https://pro.arcgis.com/en/pro-app/tool-reference/data-management/supported-notation-formats.htm:
GARS is based on longitude and latitude and divides the world into 30 minute by 30 minute cells. The first three digits represent a 30 minute wide longitude band. The fourth and fifth letters represent a 30 minute tall latitude band. Each 30 by 30 minute cell can be divided into four 15 by 15 minute cells with the 15 by 15 minute cells further divided into nine 5 by 5 minute cells.

161MG - Coordinate down to 30' x 30' cell.
161MG3 - Coordinate down to 15' x 15' cell.
161MG37 - Coordinate down to 5' x 5' cell.

The widget calculates the correct output coordinates when entering GARS 30' x 30' cell, 161MG and GARS 15' x 15' cell 161MG3, but does not calculate the correct output coordinate when entering GARS 5' x 5' cell, 161MG37

Here is the widget output for the 5' x 5' cell:
![image](https://cloud.githubusercontent.com/assets/4635490/21878026/2caa2d06-d844-11e6-8716-472ffa4ed49f.png)

Here is the correct output for the 5' x 5' cell using the ArcGIS Pro addin:
![image](https://cloud.githubusercontent.com/assets/4635490/21878047/4dc0d4fe-d844-11e6-9d86-86990e10fa30.png)

**Comment by nfeuerstein** [comment]
Document for this version.  Will address in the next.

**Comment by elinz** [comment]
@nfeuerstein @dfoll @Dbarnes1 I double-checked the widget and it is correctly converting GARS to other coordinates. The problem is that I used Pro as the baseline and it's Pro that is not working correctly. FYI. ArcMap is working.

I did validate these coordinates with GeoTrans:

- For: 161MG37 (5' x 5' cell)
GeoTrans Output: 45 2 30.0N     99 57 30.0W
Widget Output:   45° 2' 30.00"N 99° 57' 30.00"W

- For: 161MG3 (15' x 15' cell)
GeoTrans Output: 45 7 30.0N      99 52 30.0W 	
Widget Output:    45° 7' 30.00"N 99° 52' 30.00"W

- For: 161MG (30' x 30' cell)
GeoTrans Output: 45 15 0.0N      99 45 0.0W 
Widget Output:    45° 15' 0.00"N 99° 45' 0.00"W

I will close this issue and review what GARS issues I have created for Pro. I will edit existing or create new issue if necessary.
